lib/ansible/modules/crypto/get_certificate.py

```diff
@@ -0,0 +1,195 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: get_certificate
+author: "John Westcott IV"
+version_added: "2.7"
+short_description: Get a certificate from a host:port
+description:
+    - Makes an SSL connection and returns information about the presented certificate
```

Comment:
Hopefully TLS, not SSL connections.

Reply:
Hopefully... I've updated SSL to "secure".